consts/errorMessages.js

```diff
@@ -0,0 +1,8 @@
+export default {
+  friendRemove: 'Trouble removing friend, please try again.',
+  profileError: 'Trobule loading profile, please try again.',
```

Comment:
lol not a big deal but you spelled trouble wrong

Reply:
fixed